tutorials/observing_examples.ipynb

```diff
@@ -0,0 +1,372 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "T_eff = 4780 * u.K  # temperature of hatp11 [K]\n",
```

Comment:
If you obtained this value from some literature or website, please cite it. Same comment applies to other numbers used below.